.buildkite/pipeline.yml

```diff
@@ -0,0 +1,34 @@
+steps:
+  - name: "Unit"
+    command: make clean install-vendor test-ci-unit
+    plugins:
+      docker-compose#v2.5.1:
+        run: app
+  - name: "Big Unit"
+    command: make clean install-vendor test-ci-big-unit
+    plugins:
+      docker-compose#v2.5.1:
+        run: app
+  - name: ":docker: + services + metalint"
```

Comment:
nit: `":docker: + services + metalint"` -> `"Integration (Docker), Services, Tools, Metalint"` for consistency with the names of the other jobs.

Reply:
sure thing, will do.